GsaGH/Helpers/Export/Model/Lists.cs

```diff
@@ -1,3 +1,332 @@
-﻿namespace GsaGH.Helpers.Export {
-  internal class Lists { }
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using Grasshopper.Kernel;
+using GsaAPI;
+using GsaGH.Parameters;
+using OasysUnits.Units;
+
+namespace GsaGH.Helpers.Export {
+  internal class Lists {
+    internal static string GetElementList(GsaList list, ref GsaGuidDictionary<EntityList> apiLists,
+      GsaGuidDictionary<AnalysisMaterial> apiMaterials, GsaGuidDictionary<Section> apiSections,
+      GsaGuidDictionary<Prop2D> apiProp2ds, GsaGuidDictionary<Prop3D> apiProp3ds,
+      GsaGuidIntListDictionary<Element> apiElements, GsaGuidDictionary<Member> apiMembers,
+      ConcurrentDictionary<int, ConcurrentBag<int>> memberElementRelationship, GH_Component owner) {
+      if (apiLists.GuidDictionary.TryGetValue(list.Guid, out int id)) {
+        return "\"" + apiLists.ReadOnlyDictionary[id].Name + "\"";
+      }
+
+      if (list.EntityType == Parameters.EntityType.Member) {
+        list = list.Duplicate();
```

Comment:
why we need to dplicate it?

Reply:
Because this method is to get an Element list; if the method receives a Member list we want to convert that into an element list, but other places might still need the original Member list. So we duplicate the original Member list and afterwards change the type to an Element list.